pkg/gadgets/seccomp/tracer/tracer.go

```diff
@@ -23,7 +23,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-//go:generate sh -c "echo $CLANG_OS_FLAGS; GOOS=$(go env GOHOSTOS) GOARCH=$(go env GOHOSTARCH) go run github.com/cilium/ebpf/cmd/bpf2go -target bpfel -cc clang seccomp ./bpf/seccomp.c -- $CLANG_OS_FLAGS -I./bpf/ -I../../.. -target bpf -D__TARGET_ARCH_x86"
+//go:generate sh -c "echo $CLANG_OS_FLAGS; GOOS=$(go env GOHOSTOS) GOARCH=$(go env GOHOSTARCH) go run github.com/cilium/ebpf/cmd/bpf2go -target amd64 -cc clang seccomp ./bpf/seccomp.c -- $CLANG_OS_FLAGS -I./bpf/ -I../../.."
```

Comment:
Here we don't define `$CLANG_OS_FLAGS`, so that could be removed, right?

Did you forget to remove `$GOOS` and `$GOARCH` like in other gadgets?

Ditto in some other gadgets.

Reply:
You're right, I was missing this one. Thanks!

Reply:
> Ditto in some other gadgets.

Do you have a specific one? I can only see that `$CLANG_OS_FLAGS` is used in dns and sni where it's actually needed.